mod_cluster/extension/src/test/resources/org/wildfly/extension/mod_cluster/subsystem-transform-5_0_0.xml

```diff
@@ -0,0 +1,83 @@
+           balancer="${modcluster.balancer:mybalancer}"
+           listener="ajp"
+           excluded-contexts="${modcluster.excluded-contexts:contextA,contextB,contextC}"
+           flush-packets="${modcluster.flush-packets:true}"
+           flush-wait="${modcluster.flush-wait:10}"
+           load-balancing-group="${modcluster.load-balancing-group:mylbgroup}"
+           max-attempts="${modcluster.max-attempts:10}"
+           node-timeout="${modcluster.node-timeout:123}"
+           ping="${modcluster.ping:10}"
+           proxies="proxy1 proxy2"
+           proxy-url="${modcluster.proxy-url:/}"
+           session-draining-strategy="DEFAULT"
+           smax="${modcluster.smax:2}"
+           socket-timeout="${modcluster.socket-timeout:20}"
+           status-interval="99"
+           sticky-session-force="${modcluster.sticky-session-force:false}"
+           sticky-session-remove="${modcluster.sticky-session-remove:false}"
+           sticky-session="${modcluster.sticky-session:true}"
+           stop-context-timeout="${modcluster.stop-context-timeout:10}"
+           ttl="${modcluster.ttl:1}"
+           ssl-context="modcluster"
+           worker-timeout="${modcluster.worker-timeout:2}">
+        <dynamic-load-provider decay="${modcluster.dynamic-load-provider.decay:2}"
+                               history="${modcluster.dynamic-load-provider.history:10}">
+            <load-metric capacity="${modcluster.load-metric.capacity:1000.1}"
+                         type="sessions"
+                         weight="${modcluster.load-metric.weight:2}"/>
+            <load-metric capacity="512.2"
+                         type="send-traffic"
+                         weight="3"/>
+            <load-metric capacity="1024.1"
+                         type="receive-traffic"/>
+            <load-metric type="requests"
+                         weight="4"/>
+            <load-metric type="mem"/>
+            <load-metric type="cpu"/>
+            <load-metric type="heap"/>
+            <load-metric type="busyness">
+                <property name="name1"
+                          value="${property1:value1}"/>
+                <property name="name2"
+                          value="${property2:value2}"/>
+            </load-metric>
+            <custom-load-metric capacity="${modcluster.custom-load-metric.capacity:1.1}"
+                                class="SomeFakeLoadMetricClass1"
+                                module="org.wildfly.extension.mod_cluster"
+                                weight="${modcluster.custom-load-metric.weight:5}"
+            />
+            <custom-load-metric class="SomeFakeLoadMetricClass2"
+                                capacity="${modcluster.custom-load-metric.capacity:1.1}"/>
+            <custom-load-metric class="SomeFakeLoadMetricClass3"
+                                weight="${modcluster.custom-load-metric.weight:5}"/>
+        </dynamic-load-provider>
```

Comment:
@rhusar Any reason this does not have <ssl>? I see it in both what appear to be newer and older versions?

Reply:
The problem is that the legacy `ssl` configuration is mutually exclusive with `ssl-context` configuration, so it can only define one. While the newer versions support multiple configurations, we can test both within one file. Introducing a new profile to test this seems unnecessary since its tested in the newer profiles.